scipy/linalg/matfuncs.py

```diff
@@ -668,3 +669,48 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def khatri_rao(a, b):
+    """
+    Khatri-rao product
+
```

Comment:
Can you add a short paragraph here describing what the Khatri-Rao product is (in words, not math). It should be understandable and give a rough idea for users who don't yet know it.

Reply:
"A column-wise Kronecker product of two matrices"
Is this ok?

Reply:
That sounds fine to me

Reply:
For a single line function intros, you can use `Return the Khatri-Rao product of two compatible arrays` would be a nice candidate. See more about generic conventions [here](https://www.python.org/dev/peps/pep-0257/#id16)

Reply:
Adopting @ilayn's suggestion would be good.